lib/shared/src/chat/transcript/messages.ts

```diff
@@ -133,6 +133,7 @@ export interface SerializedChatMessage {
     editorState?: unknown
     speaker: 'human' | 'assistant' | 'system'
     text?: string // Changed from PromptString
+    contentPart?: ContentPart[]
```

Comment:
Not blocking. Technically, we can do `content: string | ContentPart[]` and stop using `text` everywhere. I don't mind keeping `text: string` but we should use `content` over `contentPart` because `content` is exactly that the upstream API accepts
```suggestion
    content?: string | ContentPart[]
```

Reply:
After reading the diff, I don't think we're setting `contentPart` anywhere 🤔

Reply:
removed contentPart